<a href="https://colab.research.google.com/github/jaimeisaac2020/Python-analsisis-basicos/blob/mi-github/IntConfianza_t.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **Intervalos de confianza**

# Ejemplo

Un fabricante ha inventado una nueva pólvora que fue probada en ocho proyectiles. Las velocidades resultantes en la boca del cañón, en pies por segundo, fueron las siguientes:


In [ ]:
velocidades = [3005, 2925, 2935, 2965, 2995, 3005, 2937, 2905]

Encuentre un __intervalo de confianza de _95%_ para el verdadero promedio de velocidad $\mu$__ para proyectiles de este tipo. Suponga que las velocidades en la boca del cañón están distribuidas normalmente en forma aproximada.

## Solución


Si suponemos que las velocidades $Y_{i}$ están distribuidas normalmente, el intervalo de confianza para $\mu$ es

\begin{equation}
\bar{Y}\pm t_{\alpha/2}(\frac{S}{\sqrt{n}})
\end{equation}

donde $t_{\frac{\alpha}{2}}$ está determinado por $n-1$ grados de libertad. Con los datos dados tenemos:

In [ ]:
n = len(velocidades)
import statistics
media = statistics.mean(velocidades)
desv = statistics.stdev(velocidades)
print(n)
print(media)
print(desv)

8
2959
39.08964057138413


en este ejemplo tenemos $n-1 =$ 

In [ ]:
n-1

7

grados de libertad, $1-\alpha = 0.95$, $\alpha =0.05$, $\alpha/2 = 0.025$

\begin{equation}
t_{\alpha/2} = t_{0.025},
\end{equation}

luego, $t_{0.025}=$

In [ ]:
from scipy import stats
t0_25 = stats.t(df = 7).ppf(0.975)
print(t0_25)

2.3646242510102993


entonces obtenemos

In [ ]:
import math
li = media - t0_25*(desv/math.sqrt(n))
ls = media + t0_25*(desv/math.sqrt(n))
print(li)
print(ls)

2926.3202426713874
2991.6797573286126


Es decir, nuestro intervalo de confianza del $95\%$ para el verdadero promedio de velocidad $\mu$ es

\begin{equation}
(2926.3, 2991.7)
\end{equation}

Basado en *Wackerly, D. et al. (2010). Estadística Matemática con Aplicaciones. Cengage Learning Editores, S.A. de C.V.*